dev/com.ibm.ws.microprofile.faulttolerance/src/com/ibm/ws/microprofile/faulttolerance/impl/FTConstants.java

```diff
@@ -65,4 +69,34 @@ public static double toSeconds(long nanos) {
         return ((double) nanos / (double) 1000000000);
     }
 
+    /**
+     * Format a method for display in a log message
+     * <p>
+     * Uses the format:
+     * <p>
+     * {@code com.example.MyClass.InnerClass.myMethod(String, Map, MyBean)}
+     * <p>
+     * This is a compromise between including enough information to identify the method, while not being too verbose like {@code Method.toString()}
+     *
+     * @param m the method
+     * @return a string representation of the method
+     */
+    @Trivial
+    public static String formatMethod(Method m) {
+        if (m == null) {
```

Comment:
FTConstants now has more methods than constants ... rename it/split it to FTDebug or something?